File: app/src/main/java/com/tmontovaneli/picpaychallenge/model/Payment.java
package com.tmontovaneli.picpaychallenge.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

/**
 * Created by tmontovaneli on 23/08/17.
 */

public class Payment implements Serializable {

    @JsonProperty("card_number")
    private String nrCartao;

    private Integer cvv;

    private double value;

    @JsonProperty("expiry_date")
    private String dtExpiracao;

    @JsonProperty("destination_user_id")
    private Integer idUser;

    public String getNrCartao() {
        return nrCartao;
    }

    public void setNrCartao(String nrCartao) {
        this.nrCartao = nrCartao;
    }

    public Integer getCvv() {
        return cvv;
    }

    public void setCvv(Integer cvv) {
        this.cvv = cvv;
    }

    public double getValue() {
        return value;
    }

    public void setValue(double value) {
        this.value = value;
    }

    public String getDtExpiracao() {
        return dtExpiracao;
    }

    public void setDtExpiracao(String dtExpiracao) {
        this.dtExpiracao = dtExpiracao;
    }

    public Integer getIdUser() {
        return idUser;
    }

    public void setIdUser(Integer idUser) {
        this.idUser = idUser;
    }
}

File: app/src/main/java/projetosestudos/felipe/com/trabalhe_conosco_mobile_dev/model/Payment.kt
package projetosestudos.felipe.com.trabalhe_conosco_mobile_dev.model

import com.google.gson.annotations.SerializedName

data class Payment(@SerializedName("card_number") val cardNumber: String,
                   @SerializedName("cvv") val cvv: Int,
                   @SerializedName("value") val value: Double,
                   @SerializedName("expiry_date") val expiryDate: String,
                   @SerializedName("destination_user_id") val destinationUserID: Int)